src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/InnerSourceRulesOfPlayScore.java

```diff
@@ -0,0 +1,155 @@
+package com.sap.oss.phosphor.fosstars.model.score.oss;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.HAS_INNERSOURCE_TOPIC;
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.HAS_README;
+
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.other.Utils;
+import com.sap.oss.phosphor.fosstars.model.score.FeatureBasedScore;
+import com.sap.oss.phosphor.fosstars.model.value.BooleanValue;
+import com.sap.oss.phosphor.fosstars.model.value.ScoreValue;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
```

Comment:
Please add the FEATURES  used for the score calculation as a JAVA Doc
As given here https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/BanditScore.java#L15-L23